test/models/deploy_test.rb

```diff
@@ -134,9 +134,10 @@
       deploy.commit.must_equal "abcdef"
     end
 
-    it "falls back to deploys reference" do
```

Comment:
can just remove this since we don't add test for other code that does not exist either :D